cmd/flags/flags_test.go

```diff
@@ -21,8 +21,13 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"go.uber.org/goleak"
 )
 
+func TestMain(m *testing.M) {
```

Comment:
Manually adding these functions is not a scalable approach, next time someone will surely forget to add one.

I would prefer to have a CI check, similar to `make nocover`, that will fail if some packages don't have this function (assuming we want to go with checking these leaks).